food_basic/locale/tr.txt

```diff
@@ -0,0 +1,40 @@
+# mahmutelmas06@gmail.com
+
+Wheat = Buğday	
+Flour = Un
+Potato = Patates
+Tomato = Domates
+Carrot = Havuç
+Orange = Portakal
+Milk = Süt
+Egg = Yumurta
+Cocoa Bean = Kakao Çekirdeği
+Raw meat = Çiğ et
+Venison = Geyik Eti
+Sugar = Şeker
+Chocolate Powder = Çikolata tozu
+Dark Chocolate = Bitter çikolata
+Milk Chocolate = Sütlü çikolata
+Pasta = Makarna
+Bowl = Kase
+Butter = Yağ
```

Comment:
This apparently translates back to "oil", is this correct? Translate suggests tereyağı.